examples/mergable-ingress-types/coffee-minion.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: cafe-ingress-coffee-minion
+  annotations:
+    nginx.org/mergible-ingress-type: "minion"
```

Comment:
Better to have `kubernetes.io/ingress.class: "nginx"` annotation as well in all Ingress examples